### PR TITLE
Feature#27 filter results db

### LINKED ISSUE #27: Filter Results from DB
# REQUIREMENTS
As a User i want that the map only shows me registered business with custom marks/icons that represents each kind of store

# DESIGN
image

# RECOMMENDATION 
https://developers.google.com/maps/documentation/javascript/custom-markers

### PR DESCRIPTION
**Pull request by JacquelineFM**
# Description
Bottom menu was implemented to filter results for each kind of business and showed on the map area.
Also, added info to market modal and the code from legal/index was move to market places/index.
 
 ## Screenshots 

All:
image

Products:
image
image

Services:
image
image

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -34,3 +34,27 @@ h5, h6 {
...
+.food { 
+  background-image: url( 
+'https://www.flaticon.es/svg/static/icons/svg/857/857755.svg'); 
+} 
+
+.all { 
+  background-image: url( 
+'https://www.flaticon.es/svg/static/icons/svg/684/684850.svg'); 
+}
+*/
```
Comment (KarinaDell) [inline app/assets/stylesheets/application.css]: Is this code needed? If not please delete it

**Review by EduJimenez96 — DISMISSED** [review]
I think, everything is nice 👍

**Review by KarinaDell — DISMISSED** [review]
Looks good to me

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (Ilyeo) [reply app/assets/stylesheets/application.css]: I agree with this comment above 👆

File: app/controllers/market_places_controller.rb [hunk 2]
```diff
@@ -1,12 +1,23 @@
 class MarketPlacesController < ApplicationController
-  before_action :require_login, except: %i[index, show]
+  before_action :require_login, except: [:index]
```
Comment (Ilyeo) [inline app/controllers/market_places_controller.rb:2]: could you run rubocop on this file to check offenses?

usually, rubocop suggest `%i[index]` over `[:index]`

**Review by Ilyeo — COMMENTED** [review]
looks good I left some observations

let me know if you have questions about it

**Review by EduJimenez96 — APPROVED** [review]

**Review by Ilyeo — APPROVED** [review]

**Review by KarinaDell — APPROVED** [review]

**Pull request merged**